compiler/rustc_middle/src/hir/place.rs

```diff
@@ -74,13 +74,13 @@ pub struct Projection<'tcx> {
     /// Type after the projection is being applied.
```

Comment:
Should this read "...is applied"?

Reply:
Yes, probably.
Could you fix this as well? (And squash commits into one.)